manifest/docker_list.go

```diff
@@ -0,0 +1,212 @@
+package manifest
```

Comment:
Nit: rename the file to `docker_schema2_list.go` to be consistent with other files in the package.

Reply:
Renamed it.